Documentation/building/unix-test-instructions.md

```diff
@@ -40,6 +40,46 @@ Or
 >`coreclr/Tools/dotnetcli/dotnet coreclr/Tools/MSBuild.dll /maxcpucount coreclr/tests/src/JIT/CodeGenBringUpTests/Array1.csproj /p:__BuildType=Release /p:__BuildOS=OSX`
 
 
+**Aarch64/armhf multiarch**
+
+For machines that have aarch64/armhf support, all the armhf packages will need to also be downloaded. Please note you will need to enable multiplatform support as well. Check with you distro provider or kernal options to see if this is supported. For simplicity, these instructions relate to aarch64 ubuntu enabling arm32 (hf) coreclr runs.
```

Comment:
grammar nit: "Check with you distro" => "Check with your distro"

also: kernal => kernel?